collect_app/src/main/java/org/odk/collect/android/utilities/CodeCaptureManagerFactory.kt

```diff
@@ -27,6 +27,7 @@ object CodeCaptureManagerFactory {
         return IntentIntegrator(activity)
             .setDesiredBarcodeFormats(supportedFormats)
             .setPrompt(prompt)
+            .setOrientationLocked(false) // Let UI control orientation lock
```

Comment:
I don't think we should be doing this as it locks the orientation for the activity even after whatever is using it closes (to see this hit "Configure with QR code" and then press back and rotate).

We can definitely add smarter locks in at the fragment/activity level if it feels like we need it.